diagnostic.md

```diff
@@ -5,74 +5,88 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+It defines/creates the framework of the app
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model layer
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller layer
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Not sure what is being asked here. Please advise!
```

Comment:
`Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.` does that clear it up at all?